src/monika/DebugExamples/ArrDiffEsumOsum.java

```diff
@@ -0,0 +1,43 @@
+package monika.DebugExamples;
+
+/*Program 2:  return a difference between sum of even number and odd numbers.	
+		From given numbers, count the odd and even numbers.
+		input : 10,23,26,27,88,87,65 
+	    output : evenSum - oddSum
+		
+		int getDifference(int[] input){
+		
+		}*/
+
+import java.util.Scanner;
+
+public class ArrDiffEsumOsum {
+	int evenSum, oddSum, arrDiffEsumOsum;
+
+	int getDifference(int[] input) {
+
+		for (int index = 0; index < input.length; index++) {
+			if (input[index] / 2 == 0) {
```

Comment:
When ever you came across odd and even number always use % function
if(input[index] % 2 == 0){
        even
}else{
       odd
}

Reply:
Interviewer expect % function not /